go/adbc/driver/snowflake/connection.go

```diff
@@ -777,8 +779,9 @@ func (c *cnxn) Rollback(_ context.Context) error {
 // NewStatement initializes a new statement object tied to this connection
 func (c *cnxn) NewStatement() (adbc.Statement, error) {
 	return &statement{
-		alloc: c.db.alloc,
-		cnxn:  c,
+		alloc:     c.db.alloc,
+		cnxn:      c,
+		queueSize: defaultStatementQueueSize,
```

Comment:
Is this something a user might need to configure, say for different networking conditions, or different workload profiles?

Reply:
Okay, I'll switch these to formal config params rather than using an env var